lib/src/track/local/local.dart

```diff
@@ -91,11 +91,11 @@ abstract class LocalTrack extends Track {
   /// Mutes this [LocalTrack]. This will stop the sending of track data
   /// and notify the [RemoteParticipant] with [TrackMutedEvent].
   /// Returns true if muted, false if unchanged.
-  Future<bool> mute() async {
+  Future<bool> mute(bool stopOnMute) async {
```

Comment:
I think optional parameters are needed here to avoid break changes assuming the user is using `.mute` and `.unmute` alone

`{bool stopOnMute = true}`

```suggestion
  Future<bool> mute({bool stopOnMute = true}) async {
```